runtime/doc/eval.txt

```diff
@@ -4255,8 +4255,10 @@ items({dict})						*items()*
 		order.
 
 jobclose({job}[, {stream}])				{Nvim} *jobclose()*
-		Close {job}'s {stream}, which can be one "stdin", "stdout" or
-		"stderr". If {stream} is omitted, all streams are closed.
+		Close {job}'s {stream}, which can be one "stdin", "stdout",
```

Comment:
I know you didn't change this, but this should say:

"which can be one of "stdin",..." instead of "which can be one "stdin"..."